tests/pipeline/test_site_daily_metrics.py

```diff
@@ -266,10 +288,6 @@ def test_load_for_today(self):
         for key, value in self.FIELD_VALUES.iteritems():
             assert getattr(sdm, key) == value, 'failed on key: "{}"'.format(key)
 
-
+    @pytest.mark.skip('Test stub')
     def test_no_course_overviews(self):
```

Comment:
I'm leaving this test method here because we do want to test this, but it is a lower priority